src/calculateRentalCost.js

```diff
@@ -20,7 +20,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+
+  if (days < 3) {
+    cost = days * 40;
+  } else if (days >= 3 && days < 7) {
+    cost = days * 40 - 20;
```

Comment:
Добавляй скобочки в выражения, это хорошая практика, даже не смотря на то что это простое выражение
```suggestion
    cost = (days * 40) - 20;
```